### PR TITLE
(AUTOMATIC) opensource update

### PR DESCRIPTION
**Pull request by sebsg**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -18,11 +18,17 @@ values.
 
 ## C++
 
-The C++ library (in very portable C++03) of _libaddressinput_ is used in address-related
-projects in [Chromium](http://www.chromium.org/Home).
+The C++ library (in very portable C++03) of _libaddressinput_ is the backend for
```
Comment (madmath) [inline README.md]: revert this, the existing README was cleaned up already

File: android/src/main/java/com/android/i18n/addressinput/autocomplete/gmscore/AddressAutocompleteApiImpl.java [hunk 2]
```diff
@@ -0,0 +1,107 @@
+package com.android.i18n.addressinput.autocomplete.gmscore;
```
Comment (madmath) [inline android/src/main/java/com/android/i18n/addressinput/autocomplete/gmscore/AddressAutocompleteApiImpl.java]: This file has no license and references gmscore, I wonder if it was meant to be included? Can you check?

File: README.md (hunk 1 above)
Comment (lararennie) [inline README.md]: Can you fix it internally too?

File: android/src/main/java/com/android/i18n/addressinput/autocomplete/gmscore/AddressAutocompleteApiImpl.java (hunk 2 above)
Comment (lararennie) [inline android/src/main/java/com/android/i18n/addressinput/autocomplete/gmscore/AddressAutocompleteApiImpl.java]: The corresponding unit test has the license....

But indeed if it is some of the comments look like they need sanitising.

File: README.md (hunk 1 above)
Comment (madmath) [inline README.md]: Will do.

File: android/src/main/java/com/android/i18n/addressinput/autocomplete/gmscore/AddressAutocompleteApiImpl.java [hunk 3]
```diff
@@ -0,0 +1,106 @@
+
```
Comment (sebsg) [inline android/src/main/java/com/android/i18n/addressinput/autocomplete/gmscore/AddressAutocompleteApiImpl.java]: Sorry! I put it back.

**Pull request merged**